### PR TITLE
Allow hiera config for dotfiles

### PR DESCRIPTION
**Pull request by PuppetNinja**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -81,6 +81,7 @@
   $python_virtualenvs        = { },
   $python_pyvenvs            = { },
   $python_requirements       = { },
+  $python_dotfiles           = { },
```
Comment (ghoneycutt) [inline manifests/init.pp]: this needs to be validated. Please add `validate_hash($python_dotfiles)` to the manifest.

**Comment by ghoneycutt** [comment]
Thanks @PuppetNinja 

The tests that are failing are not related to your PR. I am working on fixing them in PR #345

**Comment by ghoneycutt** [comment]
Hi @shivapoudel @stankevich 

Would one of you mind testing this PR to ensure it does what it is supposed to? I'd like to get confirmation from someone before merging.

File: manifests/init.pp (hunk 1 above)
Comment (PuppetNinja) [inline manifests/init.pp]: hash validation has been added for $python_dotfiles, thanks

**Pull request merged**